.github/workflows/composite/setup/action.yaml

```diff
@@ -17,7 +17,7 @@ runs:
         sudo pip3 install invoke semver pyyaml
         GO111MODULE="on" go get sigs.k8s.io/kind@v0.19.0
         go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.7.0
-        go install github.com/onsi/ginkgo/v2/ginkgo@v2.4.0
+        go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
From my testing, this installs the version that is in `go.mod`. For example, `go.mod` currently points to `2.17.1` and it installed `2.17.1`, even though `2.19.1` is available.

This is maybe more of an implicit pinning than the more explicit pinning that is done for a number of tools through `inv`, but I also don't have a strong opinion here. So I'm happy to follow whatever your preference is.

Reply:
so the only difference is going to be when it runs on the dev's machine, I guess we can live with that.
Then the only ask I have is, can you add a comment explaining that that go install looks at the version we have in the go.mod file?

Thanks again!

Reply:
I can add a comment in the workflow about the behavior. Would it be worth add something to `tasks.py` that checks for `ginkgo` to be installed ( I don't believe that currently happens, but I also fumble around in there a bunch so just want to confirm) and then checks it against the version that should be installed? If they're not equal, we could just drop a warning? Although not sure an additional warning would be necessary since `ginkgo` already drops the above warning.

Reply:
I'd just comment above the go install saying that it picks the version from go.mod, nothing more.
If we want to mess up with tasks.py, I'd say the best thing would do as we do with kubectl for example, keeping a local copy of the binary and using that for the tests.
That would make the dev machine identical to CI. But again, I think we can live with the comment and that's it.

Reply:
Note added.